src/components/SiteHeader/index.js

```diff
@@ -104,23 +104,31 @@ const HeaderWrapper = styled.div`
           height: 160px;
         }
         @media screen and (max-width: 539px) {
+          #header {
+            display: none;
+          }
+
           header {
-            height: 116px;
+            height: 50px;
```

Comment:
Setting this to 50px will render the error notification in the wrong position on smaller screens for unauthenticated visitors:
![Screenshot 2020-01-20 at 10 53 16](https://user-images.githubusercontent.com/1062191/72716811-25392000-3b73-11ea-9dd9-fcbef36e70af.png)

Reply:
daar ga ik even naar kijken

Reply:
gedaan